### DISCUSSION TITLE
Strange bug with diacritics

### DISCUSSION THREAD

**Discussion by reneaudet** [knowledgefutures/pubpub#1704] [Support]
Strange (specific) bug: in French we use diacritics, and diacritics on capital letters. I have a Pub with an "É" in its title. It shows well... unless in the public list of pubs (e.g. pubs associated to a collection). All other capital letters with diacritics appear correctly. Maybe it is filtered!? Thanks !

![image](https://user-images.githubusercontent.com/5399642/145450584-a4440cad-cc34-4dcf-bb70-2f5e00e350b5.png)
![image](https://user-images.githubusercontent.com/5399642/145450620-b09a3634-e19d-48be-a0cb-83eeba880c86.png)
![image](https://user-images.githubusercontent.com/5399642/145450669-0dd0ece8-196d-4683-a0bf-7307da6f3a83.png)

**Comment by gabestein** [comment]
Very odd. It looks to be a bug with the specific weight of the font we use. I'll create a ticket for our team to look at. We may need to replace the font.

<img width="238" alt="Screen Shot 2021-12-09 at 13 20 10" src="https://user-images.githubusercontent.com/639110/145453418-55c1df2a-80e6-4944-9be9-20fd66a7ac59.png">

**Reply by reneaudet** [reply]
OMG... Very strange. Thanks for the followup.

**Comment by reneaudet** [comment]
Hi Gabriel,

Any follow-up on this (specific but annoying) problem?

Thanks !

René

Le 09 12 2021 à 13:20, Gabriel Stein ***@***.******@***.***>> a écrit :



Very odd. It looks to be a bug with the specific weight of the font we use. I'll create a ticket for our team to look at. We may need to replace the font.

[Screen Shot 2021-12-09 at 13 20 10]<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F639110%2F145453418-55c1df2a-80e6-4944-9be9-20fd66a7ac59.png&data=04%7C01%7Crene.audet%40lit.ulaval.ca%7C9ee775908dc4462363dd08d9bb409585%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637746708323527569%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=WjVXpSWmEgTmoLP5pOmu2orfeA2aK4Pn6bkEIl8726s%3D&reserved=0>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpubpub%2Fpubpub%2Fdiscussions%2F1704%23discussioncomment-1781068&data=04%7C01%7Crene.audet%40lit.ulaval.ca%7C9ee775908dc4462363dd08d9bb409585%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637746708323537520%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qx8w%2FubMSw7BawcPxqysyrbXmxqIoeu9MLpt4qbnJwo%3D&reserved=0>, or unsubscribe<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FABJGIWUQEDV6SSWH2SNOS63UQDXO5ANCNFSM5JXDIPSQ&data=04%7C01%7Crene.audet%40lit.ulaval.ca%7C9ee775908dc4462363dd08d9bb409585%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637746708323537520%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=n%2B1IuT8xpBQb78PKkHmze1E5jX%2FjzU7ugEzA4ay%2Fdhc%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Crene.audet%40lit.ulaval.ca%7C9ee775908dc4462363dd08d9bb409585%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637746708323547480%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=%2BVK%2BgvOtQtXYYYpvVaHeza1keIBdGirx9smbIDUeFaE%3D&reserved=0> or Android<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Crene.audet%40lit.ulaval.ca%7C9ee775908dc4462363dd08d9bb409585%7C56778bd56a3f4bd3a26593163e4d5bfe%7C1%7C0%7C637746708323547480%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=D4%2FEFW7P2t8%2FHB%2BnuV5YKk0roc6BJUAA1vFDfQ7Tr3M%3D&reserved=0>.

**Reply by gabestein** [reply]
Hi René — we filed a bug with Adobe, which maintains the font, but have not heard back. We're working on a larger overhaul of fonts across the platform in the meantime, but it's a larger project that will take us some time, so I don't know exactly when we'll be able to resolve it. I'm sorry I don't have better news, but I'll let you know when we make progress.

**Comment by reneaudet** [comment]
Thanks for following up ! Can't wait for this new feature about fonts :)

René

Le 14 04 2022 à 17:32, Gabriel Stein ***@***.******@***.***>> a écrit :



Hi René — we filed a bug with Adobe, which maintains the font, but have not heard back. We're working on a larger overhaul of fonts across the platform in the meantime, but it's a larger project that will take us some time, so I don't know exactly when we'll be able to resolve it. I'm sorry I don't have better news, but I'll let you know when we make progress.
